packages/perseus/src/widgets/grapher/util.tsx

```diff
@@ -479,34 +511,46 @@ const Logarithm = _.extend({}, PlotDefaults, {
 
     allowReflectOverAsymptote: true,
 
-    getCoefficients: function (coords, asymptote) {
+    getCoefficients: function (
+        coords: Coords,
+        asymptote: Coords,
+    ): ReadonlyArray<number> | undefined {
         // It's easiest to calculate the logarithm's coefficients by thinking
         // about it as the inverse of the exponential, so we flip x and y and
         // perform some algebra on the coefficients. This also unifies the
         // logic between the two 'models'.
-        const flip = (coord: any) => [coord[1], coord[0]];
+        const flip = (coord: Coord): Coord => [coord[1], coord[0]];
         const inverseCoeffs = Exponential.getCoefficients(
-            _.map(coords, flip),
-            _.map(asymptote, flip),
+            _.map(coords, flip) as Coords,
+            _.map(asymptote, flip) as Coords,
         );
-        const c = -inverseCoeffs[2] / inverseCoeffs[0];
-        const b = 1 / inverseCoeffs[0];
-        const a = 1 / inverseCoeffs[1];
-        return [a, b, c];
+        if (inverseCoeffs) {
+            const c = -inverseCoeffs[2] / inverseCoeffs[0];
+            const b = 1 / inverseCoeffs[0];
+            const a = 1 / inverseCoeffs[1];
+            return [a, b, c];
+        }
```

Comment:
Added this to an if statement because of TS errors stating `inverseCoeffs` could be undefined.